app/app.rb

```diff
@@ -34,6 +34,17 @@
   body {}
 end
 
+get '/devotionals' do
+  devotionals = Devotional.all
+
+  content_type :json
+  devotionals.to_json
+
+rescue KeyError
```

Comment:
Under what circumstances would we get a KeyError?

Reply:
Oof so I guess I can't have a KeyError with an array. I'll take a shot at fixing this and see what you think.

Reply:
Let me know if you want to chat through it